tests/slow/src/test/scala/tests/sbt/SbtLspSuite.scala

```diff
@@ -419,4 +420,18 @@ class SbtLspSuite extends BaseImportSuite("sbt-import") {
       )
     } yield ()
   }
+
+  test("min-sbt-version") {
```

Comment:
This is just in case, Bloop will be dropping the support for sbt `0.13`, so this test should fail if anyone wants to update to that Bloop version. We will need to freeze Bloop version for sbt 0.13 in that case

Reply:
At some point Metals should probably drop 0.13 as well. The times are ripe, I think